starwars/src/App.js

```diff
@@ -31,8 +32,14 @@ class App extends Component {
 
   render() {
     return (
+
       <div className="App">
         <h1 className="Header">React Wars</h1>
+        <List  height={this.state.starwarsChars}/>
+
+        <CharButton nameprops = {this.state.starwarsChars} />
```

Comment:
One of the best things I think you can do is use some different naming conventions. CharButton is ok as a component name here IF your intention is to make it button-like/clickable in the future. Otherwise, something like `CharacterContainer` or `Characters` might be more appropriate. Those are just options, though. You could use another name as well.

For your prop name `nameprops`, it would be better to use a naming convention that fits the data better. So, if `this.state.starwarsChars` is a list of characters, you could choose a prop name like `starwarsChars`, `characters`, `chars`, `charsList`, etc. Whichever makes most sense to you. Those names fit the description of the data better. Good naming conventions will help others read your code and you. :)